app/views/posts/index.html.erb

```diff
@@ -1 +1,34 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<%= render "shared/header" %>
+<div class="posts-container">
+ <div class="posts-header">
+  <img src="<%= @user.photo %>" alt="user photo">
+  <div>
+     <h2>Name: <%= @user.name %></h2>
+     <h4>Number of posts: <%= @user.post_counter %></h4>
+  </div>
+</div>
+ <% @posts.each do |post| %>
+ <div class="all-posts-container">
+    <div class="all-posts">
+      <h2><%= post.title %></h2>
+      <p><%= post.text %></p>
+      <div>
+         <span>Comments: <%= post.comments_counter %></span>
+         <span>Likes: <%= post.likes_counter %></span>
+         <h5>
+         <%= link_to "Read post", user_post_path(post) %>
+         </h5>
+     </div>
+    </div>  
+ </div>
+  <h1 class="comments-header">All comments</h1>
+  <div class="post-comments">
+   <% post.comments.each do |comment| %>
+      <h4>Username: <%= comment.user.name %> - <span> <%= comment.text %> </span> </h4>
+   <% end %>
+  </div>
+ <% end %>
+</div>
+<button type="button">Pagination</button>
```

Comment:
- [ ] As per project requirements, please make sure to use the recent_comments method to render the most recent 5 comments on the posts index page. All the comments should only appear on the post show page. 👍